docker-compose.test.yml

```diff
@@ -1,30 +1,49 @@
 version: "2"
 services:
-  debug:
-    image: busybox
-    command: sh
+  test-data-generator:
+    build:
+      context: .
+      dockerfile: Dockerfile.test-data-generator
+    entrypoint: /run-test.sh
+    command:
+      - create_test_files
+      # up / run docker-compose with "wait_after_create_test_files=yes" to make test-data-generator wait; this is to
+      # emulate a long running data-generation container, from which gpg-s3sync will perform backup
+      - ${wait_after_create_test_files}
     volumes:
       - data:/data/plain
-      - encrypted-data:/data/encrypted
+      - ./run-test.sh:/run-test.sh
```

Comment:
No .. it is ignored in `.dockerignore` Line 4, for the sake of the main `Dockerfile`